samples/client/petstore/lua/petstore/api/pet_api.lua

```diff
@@ -18,6 +18,7 @@ local basexx = require "basexx"
 -- model import
 local petstore_api_response = require "petstore.model.api_response"
 local petstore_pet = require "petstore.model.pet"
+local petstore_pet = require "petstore.model.pet"
```

Comment:
This seems to be an error.

Reply:
UPDATE: I've filed 
https://github.com/OpenAPITools/openapi-generator/pull/1414 to fix the issue (duplicated imports)